04/app.js

```diff
@@ -31,10 +42,22 @@ function getChannelColor(colorInHex, channelName) {
             break;
     }
 
+    // console.log(colorInHex);
     const channelColorHex = colorInHex.substr(start, 2);
     const channelColorDec = parseInt(channelColorHex, 16);
 
     return channelColorDec; 
 }
-
-
+//DZIAŁA
+function getColor(e) {
+    console.log(e.target.value);
+    const boxElement = document.querySelector('.box');
+    setBoxShadow(boxElement, e.target.value);
+}
+//NIE DZIAŁA ustawienie cienia
+function setShadow(e) {
+    console.log(e.target.value);
+    const boxElement = document.querySelector('.box');
+    const opacityValue = e.target.value / 100;
+    setBoxShadow(boxElement, opacityValue);
```

Comment:
Zwróć uwagę, jak wygląda deklaracja dla funkcji setBoxShadow (linia 19).
Drugi parametr to kolor w formacie #000000, a dopiero trzeci to przeźroczystość - musisz trzymać się tej konwencji, aby to zadziałało.

Reply:
ok zrobione